frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/AttachmentPortletUtils.java

```diff
@@ -129,6 +129,14 @@ private void serveAttachmentBundle(List<AttachmentContent> attachments, Resource
         if(attachments.size() == 1){
             filename = attachments.get(0).getFilename();
             contentType = attachments.get(0).getContentType();
+            if (contentType.equalsIgnoreCase("application/gzip")) ;
```

Comment:
is this if-statement in correct syntax? Is it correct to have semicolon in the end?

Reply:
Definitely a mistake, but I'm not sure whether it will cause any wrong behavior. Should be corrected anyways